include/swift/Option/Options.td

```diff
@@ -534,10 +534,20 @@ def Rpass_missed_EQ : Joined<["-"], "Rpass-missed=">,
 
 def save_optimization_record : Flag<["-"], "save-optimization-record">,
   Flags<[FrontendOption]>, HelpText<"Generate a YAML optimization record file">;
```

Comment:
This needs an update.  Can you please also summarize the command line options in the description?

Reply:
I think this should stay as is, since `-save-optimization-record` without `=<format>` defaults to YAML. I'll update the description.